allure-testng-adaptor/src/test/java/ru/yandex/qatools/allure/testng/AllureTestListenerSuiteNameTest.java

```diff
@@ -41,7 +41,7 @@ public void setUp() throws IOException {
 
         List<String> suites = new ArrayList<>();
         URL resource = getClass().getClassLoader().getResource("suite3.xml");
-        assumeNotNull(resource);
+        assertNotNull("could not find suite3.xml", resource);
```

Comment:
you should not use asserts in before. For this case it should be separate testcase